bee-api/bee-rest-api/src/routes/api/core/v2/info.rs

```diff
@@ -75,7 +80,7 @@ async fn info<B: StorageBackend>(Extension(args): Extension<ApiArgsFullNode<B>>)
                 v_byte_factor_key: args.protocol_config.rent().v_byte_factor_key,
                 v_byte_factor_data: args.protocol_config.rent().v_byte_factor_data,
             },
-            token_supply: TOKEN_SUPPLY.to_string(),
+            token_supply: protocol_parameters.token_supply().to_string(),
```

Comment:
Why do we use `args.protocol_config` for some and for others `protocol_parameters`?

Reply:
Because the supply is not in the protocol config. At some point I will remove the protocol config because this is all coming in a ProtocolParameters object in the snapshot. For now I only replace what is not in the current protocol config. I can change more like the one you commented but overall the API won't work until I finish the snapshot PR.